### DISCUSSION TITLE
Docker: Nominatim search query not work

### DISCUSSION THREAD

**Discussion by Tony763** [osm-search/Nominatim#2226] [Q&A] [answered]
Hello, I would like to ask about Nominatim in docker and search query. My installation works quite well in manner of creating trips by positions (lon, lat), but If I want to get position from address by sending a search query, I got error 400.

This works:
`https://nominatim.openstreetmap.org/search?q=Praha+4+-+Bráník&format=jsonv2`

This not:
`http://127.0.0.1:8083/search?q=Praha+4+-+Bráník&format=jsonv2`

Nominatim is latest version from dockerhub.

**Comment by lonvia** [comment, marked as answer]
We don't provide the docker images and can't help you there. You should ask whoever has created the image.
